torch/csrc/jit/export.cpp

```diff
@@ -25,6 +25,7 @@ std::string value_name(Value* n) {
 struct ExportContext {
   size_t num_blocks = 0;
   bool export_raw_ir = false;
+  bool aten_fallback = false;
```

Comment:
time to make it an enum - too many bool fields flying around.

Basically we have:
- raw
- onnx strict
- onnx with Aten ops everywhere (added by Lu)
- onnx with Aten ops as fallback

I think export mode would make it much saner - with your diff we now have 3 bool fields :( And it's pretty confusing for this code maintainability